host/CMakeLists.txt

```diff
@@ -112,19 +112,19 @@ target_link_libraries(oehost PUBLIC oe_includes)
 # TODO: Replace these with `find_package` and add as dependencies to
 # the CMake package.
 if (UNIX)
-  find_library(CRYPTO_LIB NAMES crypto)
-  add_library(crypto SHARED IMPORTED)
-  set_target_properties(crypto PROPERTIES IMPORTED_LOCATION ${CRYPTO_LIB})
+  find_library(CRYPTO_LIB NAMES cryptopp)
```

Comment:
I'm sorry, I'm lost, why did we change the library from crypto to cryptopp?

If there isn't a ready made CMake module for finding `crypto` that's fine, the only thing we're really trying to get out of this is a configuration time check for the `crypto` library (right now it's link-time, because we don't use `find_library` with `REQUIRED` nor are we using a dedicated CMake module for it). Whatever we use we need to duplicate for our installed CMake package too, which is why a simple module which finds the crypto library we're expecting would be idea, but this PR actually looks like it's changing the library to this different cryptopp implementation instead?